.buildkite/pipeline.yml

```diff
@@ -1,9 +1,9 @@
 # Nodes with values to reuse in the pipeline.
 common_params:
   plugins: &common_plugins
-    - automattic/a8c-ci-toolkit#2.13.0
+    - automattic/a8c-ci-toolkit#2.17.0
```

Comment:
Did you consider upgrading to the [latest version, 3.0.1](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/releases/tag/3.0.1)?

Reply:
Update: As a matter of fact, I think the `--patch-cocoapods` flag used below was introduced in that version, meaning we do need to be on it.

The fact that CI passes is, I think, a false positive. WordPressUI has no dependencies and therefore does not run into the low deployment target issue that other libraries experienced.

Reply:
I did, but [the build failed](https://buildkite.com/automattic/wordpress-ui-ios/builds/148#_), so I reverted 🙈 

I'll take another run at it